todo_app/app/controllers/todos_controller.rb

```diff
@@ -0,0 +1,17 @@
+class TodosController < ApplicationController
+
+def index
+  @todos = Todo.all
+end
+
+def new
+  @todo = Todo.new
+
+  redirect_to todos_path(@todo)
```

Comment:
If you're redirecting to the index, you shouldn't pass in an argument to the path helper.

Did you mean to write out `todo_path(@todo)`? Because in that case, you would need to pass `@todo` in as an argument.